deploy/gk-deploy

```diff
@@ -256,6 +256,9 @@ check() {
   if [[ ${cond} =~ $number_re ]]; then
     wait_limit=${cond}
   fi
+  if [[ "${resource}" == *s ]]; then
```

Comment:
It is weird, I agree, but I couldn't think of a better way to implement it without making the argument parsing (of a helper function!) even more convoluted than it already is. And I just realized I didn't document this function in an earlier patch like I meant to, so I'll do that here.

Also, for some reason globs in bash tests don't get quoted. I don't entirely understand why, but I think it breaks if you quote them.